services/ingestion/etl/inforegio/xml/src/lib/transform.js

```diff
@@ -0,0 +1,150 @@
+/*
+ * Transform message (INFOREGIO XML)
+ */
+
+// Check if field is an array or a sting.
+const checkData = data => {
+  if (data && typeof data === 'object') {
+    return data[0];
+  } else if (data && typeof data === 'string') {
+    return data;
+  }
+  return '';
+};
+
+// Formats date from DD/MM/YYYY to ISO 8601 date format.
+const formatDate = date => {
+  if (!date) return null;
+  const d = date.toString().split(/\//);
+  if (d === null || d.length !== 3) return null;
+  if (d[2].length === 2) d[2] = `20${d[2]}`;
+  return new Date(d[2], d[1] - 1, d[0]).toISOString();
+};
+
+// Get and format adress from different fields.
+const getAddress = record => {
+  let address = '';
+  if (record.Beneficiary_address) {
+    address += checkData(record.Beneficiary_address);
+  }
+  if (record.Beneficiary_Post_Code) {
+    if (address !== '') address += ', ';
+    address += checkData(record.Beneficiary_Post_Code);
+  }
+  if (record.Beneficiary_City) {
+    if (address !== '') address += ', ';
+    address += checkData(record.Beneficiary_City);
+  }
+  return address;
+};
+
+const formatBudget = budget => {
+  if (!budget) return null;
+  const b = budget.toString().split(' ');
```

Comment:
`budget.toString().split(' ').slice(1).join('')` should do the job just as well, no need for extra loops